packages/maumau-server/src/game/game-state.ts

```diff
@@ -6,10 +6,12 @@ import shuffle from '../utils/shuffle';
 
 import { autoAcceptSevens } from './listeners/auto-accept-seven';
 import { doKannetIfOnlyOption } from './listeners/do-kannet-if-only-option';
-import { hasGameEnded } from './listeners/has-game-ended';
 import { reducer, State, Action } from './reducer';
 
-const AMOUNT_OF_CARD_PER_PLAYER = 7;
+let AMOUNT_OF_CARD_PER_PLAYER: number[] = [];
+AMOUNT_OF_CARD_PER_PLAYER[2] = 7;
+AMOUNT_OF_CARD_PER_PLAYER[3] = 7;
+AMOUNT_OF_CARD_PER_PLAYER[4] = 6;
```

Comment:
Do you know a nicer way to map this? I want to map `number of players` to `cards per player`

Reply:
we could do it this way:
```
const mapping: Record<number, number> = {
    4: 7,
    2: 7,
    3:7,
    4:6,
}
```

Reply:
Done